operators/StartPostgresqlOperators.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const PostgresqlOperator = require('./postgresql-operator/PostgresqlOperator');
+const PostgresqlBindOperator = require('./postgresql-operator/PostgresqlBindOperator');
+
+const postgresqlOperator = new PostgresqlOperator();
+const postgresqlBindOperator = new PostgresqlBindOperator();
+postgresqlOperator.init();
+postgresqlBindOperator.init();
```

Comment:
@ketakig12  
The idea of naming operator on generic service name is to use it for providing other service features(apart from multi-tenancy). As of now, we are using postgresql-operator only for provisioning multi-tenancy but in future, we may need to include other functionality also. That's why we did not named it as logicalDBOperator or similar.
Reg. name of CRDS, since resource created is a database and to differentiate it from dedicated database we are naming it as logicaldb.

Reply:
AFAIK, there is no convention regarding the CRD and corresponding operator class in K8s, but with respect to Service Fabrik, as the code base is part of it, there should be some consistency. I would say, let's not name it `PostgresqlOperator`. Because in this, it is generic enough and the provisioning type is not understood. You can put it as `PostgresqlMTOperator` and the CRD also then can be `postgresqlmt` rather than `logicaldb`